project/src/main/puzzle/Puzzle.tscn

```diff
@@ -827,45 +826,43 @@ codes = [ "delays", "bigfps", "finish" ]
 [node name="PuzzleMusicManager" type="Node" parent="."]
 script = ExtResource( 19 )
 
-[node name="SceneTransitionCover" parent="." instance=ExtResource( 50 )]
-
+[connection signal="cheat_detected" from="CheatCodeDetector" to="." method="_on_CheatCodeDetector_cheat_detected"]
+[connection signal="cheat_detected" from="CheatCodeDetector" to="Hud/PuzzleTrace" method="_on_CheatCodeDetector_cheat_detected"]
+[connection signal="cheat_detected" from="CheatCodeDetector" to="Hud/VBoxContainer/FpsLabel" method="_on_CheatCodeDetector_cheat_detected"]
+[connection signal="cheat_detected" from="CheatCodeDetector" to="Hud/VBoxContainer/FrameDropsLabel" method="_on_CheatCodeDetector_cheat_detected"]
+[connection signal="timeout" from="Fg/FoodItems/FoodFlightTimer" to="Fg/FoodItems" method="_on_FoodFlightTimer_timeout"]
+[connection signal="hit_next_pieces" from="Fg/GoopGlobs" to="GoopSfx" method="_on_GoopGlobs_hit_next_pieces"]
+[connection signal="hit_playfield" from="Fg/GoopGlobs" to="Fg/Playfield" method="_on_GoopGlobs_hit_playfield"]
+[connection signal="hit_playfield" from="Fg/GoopGlobs" to="GoopSfx" method="_on_GoopGlobs_hit_playfield"]
+[connection signal="hit_wall" from="Fg/GoopGlobs" to="Fg/WallGlobViewports" method="_on_GoopGlobs_hit_wall"]
+[connection signal="hit_wall" from="Fg/GoopGlobs" to="GoopSfx" method="_on_GoopGlobs_hit_wall"]
+[connection signal="finished_spin_move" from="Fg/PieceManager" to="Fg/Bursts" method="_on_PieceManager_finished_spin_move"]
+[connection signal="finished_squish_move" from="Fg/PieceManager" to="Fg/Bursts" method="_on_PieceManager_finished_squish_move"]
+[connection signal="squish_moved" from="Fg/PieceManager" to="Fg/GoopGlobs" method="_on_PieceManager_squish_moved"]
```

Comment:
Can we order these connections in a separate merge? It makes the commit logs look busy.

Reply:
Reordered connections in 56b911c57e85efef12ca87fdc47435ddd434cfad; changelog now looks simple.